web/packages/extension/src/utils.ts

```diff
@@ -122,9 +122,9 @@ if (typeof chrome !== "undefined") {
     runtime = chrome.runtime;
 
     openOptionsPage = () =>
-        promisify((cb: () => void) =>
+        promisify((cb) =>
             chrome.tabs.create({ url: "/options.html" }, cb)
-        );
+        ).then(() => window.close());
```

Comment:
I'd rather move the `window.close()` to the caller of `utils.openOptionsPage()` (`popup.ts`), since this should be just a low-level compatibilily layer for extensions API.